src/forms/tests/manual/forms.html

```diff
@@ -397,5 +397,15 @@ <h2>Buttons</h2>
         <input type="button" class="pure-button pure-button-primary" value="Input Button" />
         <input type="reset" class="pure-button pure-button-primary" value="Reset" />
     </form>
+
+
+    <h2>File Uploads</h2>
+
+    <form class="pure-form">
+        <label class="pure-button" tabindex="0">
+            Browse <input type="file" hidden>
+        </label>
+        <input type="reset" class="pure-button pure-button-primary" value="Reset" />
```

Comment:
The only problem with the solution was that when you put another button next to it, the upload button was pushed down a little by the margin. I added this button so you can visually see what I mean.

Reply:
Doesn't seem too bad to me, what does it look like in other browsers?

Reply:
Here's Safari:
<img width="196" alt="safari" src="https://user-images.githubusercontent.com/1103622/92414401-b1f01880-f119-11ea-8a60-287551854bb9.png">

And Firefox:
<img width="194" alt="firefox" src="https://user-images.githubusercontent.com/1103622/92414405-b74d6300-f119-11ea-9ede-e85eee117dcc.png">

I'm on Mac, so I don't have IE/Edge